capstone/fitness/src/main/webapp/profile-info.html

```diff
@@ -41,6 +41,13 @@
         <br>
         <br>
 
+        <div class="input-container">
+          <label for="mileTime">How fast can you run you a mile in minutes (it's okay to estimate)</label>
```

Comment:
run a mile?

Reply:
Yes, for Pierce's feature he needs to know the time in which they can run one mile.